sklearn/neighbors/tests/test_ball_tree.py

```diff
@@ -101,3 +101,45 @@ def one_arg_func(x):
     msg = "takes 1 positional argument but 2 were given"
     with pytest.raises(TypeError, match=msg):
         BallTree(X, metric=one_arg_func)
+
+
+METRICS_TO_TEST = ["euclidean", "manhattan", "minkowski", "chebyshev"]
```

Comment:
I think this list can be renamed `PARAMETERLESS_METRICS` be moved at the top of the file. What do you think?

```suggestion
PARAMETERLESS_METRICS = [metric_name for metric_name, params in METRICS.items() if len(params) == 0]
```

Reply:
Over here minkowski is not parameterless. I did not include the others because they use rand defined at the top which conflicts with the random in the test.